symphony-app-authentication-api/pom.xml

```diff
@@ -28,6 +28,10 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
```

Comment:
Add the scope test.

Reply:
This scope has already been defined in the dependency management.